src/kit/Progress.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+
+import { floatToPercent } from 'kit/internal/string';
+import { ShirtSize } from 'kit/Theme';
+import Tooltip from 'kit/Tooltip';
+
+import css from './Progress.module.scss';
+
+export interface BarPart {
+  bordered?: string;
+  color: string; // css color
+  label?: string;
+  percent: number; // between 0-1
+}
+
+export interface Props {
+  inline?: boolean;
+  parts: BarPart[];
+  showLegend?: boolean;
+  size?: ShirtSize;
+  title?: string;
+}
+
+const partStyle = (part: BarPart) => {
+  let style = {
+    backgroundColor: part.color,
+    borderColor: 'var(--theme-float-border)',
+    borderStyle: 'none',
+    borderWidth: 1,
+    width: floatToPercent(part.percent, 0),
+  };
+
+  if (part.bordered) {
+    style = { ...style, borderStyle: 'dashed' };
+  }
```

Comment:
ditto

Reply:
I'm considering removing this `part.bordered` option? It's used only for the Potential section of the SlotAllocationBar